.github/workflows/CI.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on:
+  pull_request:
+    paths:
+      - '**.scala'
+      - '**.java'
+      - '**.sbt'
+      - '.github/workflows/**.yml'
+  push:
+    branches:
+      - master
+    paths:
+      - '**.scala'
+      - '**.java'
+      - '**.sbt'
+      - '.github/workflows/**.yml'
+
+jobs:
+  code_format:
+    name: Code Format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: jcheckstyle
+        run: ./sbt jcheckStyle
+  test_jdk11:
+    name: Test JDK11
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v7
+        with:
+          java-version: adopt@1.11
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache
+          key: ${{ runner.os }}-jdk11-${{ hashFiles('**/*.sbt') }}
+          restore-keys: ${{ runner.os }}-jdk11-
```

Comment:
Is this key correct?

Reply:
Yes. It has fallback mechanism https://circleci.com/docs/2.0/caching/#example-caching-configuration

Reply:
I read below document after I wrote above comment. And though I understood the specification of `restore-keys`, I forgot removing it.
https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows